.pre-commit-config.yaml

```diff
@@ -117,44 +133,93 @@ repos:
           - --config=.bandit.yml
   # Run bandit on everything except the "tests" tree
   - repo: https://github.com/PyCQA/bandit
-    rev: 1.7.5
+    rev: 1.7.10
```

Comment:
This version of bandit should match that on line 127.

Reply:
Please see https://github.com/cisagov/skeleton-docker/pull/246.